reconcile/response.txt

```diff
@@ -0,0 +1,7 @@
+Not trying to engage on the debate, I accept the decision. Just pointing out where I don't agree:
```

Comment:
What is this file about? :sweat_smile:

Reply:
error 😂

Reply:
It was a text file with arguments for a discussion 😂 

removed